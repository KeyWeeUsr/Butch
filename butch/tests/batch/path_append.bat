path space
echo %errorlevel%
echo %PATH%
echo %errorlevel%
path %PATH%middle%PATH%
echo %errorlevel%
echo %PATH%
echo %errorlevel%
