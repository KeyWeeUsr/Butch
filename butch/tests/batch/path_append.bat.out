0
space
0
0
spacemiddlespace
0
